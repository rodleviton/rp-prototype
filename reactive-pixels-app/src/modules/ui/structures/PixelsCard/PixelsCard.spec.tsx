import * as React from 'react';
import { render } from 'react-testing-library';
import WithRouter from '../../utils/test/WithRouter';
import WithTheme from '../../utils/test/WithTheme';
import { PixelsCard } from './';

describe('<PixelsCard />', () => {
  it('should render <PixelsCard /> component', () => {
    const { container } = render(
      <WithRouter>
        <WithTheme>
          <PixelsCard />
        </WithTheme>
      </WithRouter>,
    );

    expect(container.firstChild).toMatchSnapshot();
  });
});
