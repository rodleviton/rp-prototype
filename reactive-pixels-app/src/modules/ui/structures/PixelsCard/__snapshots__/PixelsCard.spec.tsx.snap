// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PixelsCard /> should render <PixelsCard /> component 1`] = `
<div
  class="css-1x10l6g"
>
  <div
    class="css-1lz8w4n"
  />
  <div
    class="css-pvvspc"
  />
</div>
`;
